Makefile

```diff
@@ -26,10 +26,10 @@ docker-compose-build:
 
 .PHONY: review
 review: test-cluster ## Specify review AKS environment
-	# PULL_REQUEST_NUMBER is set by the GitHub action
-	$(if $(PULL_REQUEST_NUMBER), , $(error Missing environment variable "PULL_REQUEST_NUMBER"))
+	# PR_NUMBER is set by the GitHub action
+	$(if $(PR_NUMBER), , $(error Missing environment variable "PR_NUMBER"))
 	$(eval include global_config/review.sh)
-	$(eval export TF_VAR_pull_request_number=-$(PULL_REQUEST_NUMBER))
+	$(eval export TF_VAR_PR_NUMBER=-$(PR_NUMBER))
```

Comment:
it's a terraform var, and I don't see any change in terraform?
or maybe '$(eval export TF_VAR_pull_request_number=-$(PR_NUMBER))' instead?